src/models/async/RevAiJobOptions.ts

```diff
@@ -23,6 +24,7 @@ export interface RevAiJobOptions {
     skip_diarization?: boolean;
     skip_punctuation?: boolean;
     skip_postprocessing?: boolean;
+    diarization_type?: DiarizationType;
```

Comment:
I think this should be a string type like `transcriber`

Reply:
can you please explain why it should be a string if we define an enum? not an expert in nodejs/typescript

Reply:
It's a matter of responsibility. We can use the API to enforce which options are available without having to update the SDK.

Reply:
Also, it follows the pattern of transcriber

Reply:
fixed